testfiles/WWHPSimpleAuto.idf

```diff
@@ -26,7 +26,7 @@
 !           tries to meet the load.
 ! SolDis=FullInteriorAndExterior, Aniso, Detailed Interior and Exterior Convection
 
-  Version,8.7;
+  Version,8.6;
```

Comment:
@Myoldmopar I ran this again, supposedly make-pretty transitioning to 8.8, which is what I'm seeing everywhere else. Is there any reason this file should be moved back to 8.6?